src/components/Button/Base.js

```diff
@@ -25,9 +34,11 @@ const Button = ({ variant, size, children, ...rest }) => {
   }
 
   return (
-    <StyledButton variant={variant} size={size} {...rest}>
-      {children}
-    </StyledButton>
+    <ThemeProvider theme={defaultTheme}>
```

Comment:
Do we need a `ThemeProvider` wrapping each instance of the `Button`? I was under the impression that it should be used as a singleton.

Reply:
All it does is providing a context. And you are right we only need one provider for the application. This one provides a default theme. To be honest I am not sure if need a default theme. That's why I created this PR to get some feedback.

Reply:
Maybe we can do without this for now, but implement a `ThemeProvider` in the `Button` examples to fix that page? I'd like Oscar to weigh in once he's back from vacation

Reply:
Sure.

Reply:
Thanks!